src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -539,6 +540,7 @@ MulticopterAttitudeControl::parameters_update()
 	param_get(_params_handles.vtol_wv_yaw_rate_scale, &_params.vtol_wv_yaw_rate_scale);
 
 	_actuators_0_circuit_breaker_enabled = circuit_breaker_enabled("CBRK_RATE_CTRL", CBRK_RATE_CTRL_KEY);
+	_termination_enabled = !circuit_breaker_enabled("CBRK_FLIGHTTERM", CBRK_FLIGHTTERM_KEY);
```

Comment:
This should all be handled in the commander. And the output (fmu, px4io) drivers need to handle the `armed.lockdown` condition.